src/main/java/org/ecocean/api/UserHome.java

```diff
@@ -14,7 +14,6 @@
 import org.ecocean.Annotation;
 import org.ecocean.Encounter;
 import org.ecocean.ia.Task;
```

Comment:
UserHome does _not_ need to be modified. it should be left with the projects section as-is.